Sprint-3/implement/rotate-char.js

```diff
@@ -6,6 +6,28 @@
 // This function is commonly used for text encryption and decryption,
 // where shifting characters by a certain value can obscure their meaning or reveal hidden messages.
 
+
+//......Answer......
+
+const rotateCharacter = (char, shift) => {
+    const alphabet = 'abcdefghijklmnopqrstuvwxyz';
+    const ALPHABET = 'ABCDEFGHIJKLMNOPQRSTUVWXYZ';
```

Comment:
Do you need a lowercase and uppercase array here? Is there a string method that can do this for you?

Reply:
I have updated it to String method.